src/Stratis.Bitcoin.Features.BlockStore.Tests/LoopTests/BlockStoreLoopStepBaseTest.cs

```diff
@@ -35,6 +36,7 @@ internal class FluentBlockStoreLoop : IDisposable
         private DataFolder dataFolder;
         private Mock<INodeLifetime> nodeLifeTime;
         private Mock<ILoggerFactory> loggerFactory;
+        private IBlockDownloadState blockDownloadState;
```

Comment:
IBD states for Initial Block Download, I'd prefer to preserve "Initial" in the name

Reply:
Fixed